packages/curvenote-cli/src/session/session.ts

```diff
@@ -514,6 +514,10 @@ export async function getSession(
   let session;
   try {
     session = await Session.create(data.current, { logger });
+    if (data.environment) {
+      logger.warn('Checking user token...');
+      await checkUserTokenStatus(session);
+    }
```

Comment:
This will now show up on CI and let us understand any token issues in the actions